### ISSUE TITLE
iphone/ipad fullscreen capabilities not detected

### ISSUE THREAD

**Issue by paolobenve** [Modernizr/Modernizr#2242]
I suppose that modern iphone/ipad support fullscreen, but Modernizr doesn't detect it

**Comment by patrickkettner** [comment]
@paolobenve which fullscreen are you referring to - how does a developer or user get into that mode

**Comment by paolobenve** [comment]
`if (Modernizr.fullscreen)`

**Comment by patrickkettner** [comment]
Not the modernizr feature. What are you doing on the device to get into fullscreen mode

**Comment by paolobenve** [comment]
no, my problem is that `Modernizr.fullscreen` gives me false on iphone/ipad.

Is it correct?

**Comment by paolobenve** [comment]
Let me ask the question in another way: since `Modernizr.fullscreen` doesn't detect fullscreen capabilities on ipad/iphone, is there a way to detect them on those devices?

**Comment by patrickkettner** [comment]
We are not communicating well. 

I understand that modernizr is failing to detect fullscreen for you. I want to fix that. In order to fix that I need to understand how to replicate the failure properly. In ordet for me to be able to do that, I need you to explain to me the steps you are following on your ipad/iPhone that puts your ipad/iPhone into fullscreen mode so that I can fix modernizr to detect that mode.

**Comment by roblarsen** [comment]
> no, my problem is that Modernizr.fullscreen gives me false on iphone/ipad.

> Is it expected?

[Can I use? Fullscreen API](http://caniuse.com/fullscreen/embed/)

**Comment by paolobenve** [comment]
@roblarsen, but how is possible that they didn't implemented the fullscreen api on ios safari? do they use any alternative method to go fullscreen?

@patrickkettner, `elem.webkitRequestFullScreen(Element.ALLOW_KEYBOARD_INPUT)` is used to go fullscreen

**Comment by roblarsen** [comment]
@paolobenve I don't know, today is the first time I looked at the issue. It looks like [it was intentionall disabled several years ago](https://bugs.webkit.org/show_bug.cgi?id=108250)

**Comment by patrickkettner** [comment]
@paolobenve you said 

> but I cannot verify whether it actually goes fullscreen

If that's the case, I am confused as to why you opened the issue? What made you feel that it should be `true`?
